### PR TITLE
test: Delays to fix flakes caused when getting ID tokens

### PR DESCRIPTION
**Pull request by ealemayehu**

### CODE REVIEW

File: main/integration-tests/support/setup.js [hunk 1]
```diff
@@ -131,6 +136,10 @@ class Setup {
       userRole: 'researcher',
       projectId,
     });
+
+    // Delay to prevent eventual consistency issues from affecting the next call.
+    await sleep(1000);
+
```
Comment (hatimkhan) [inline main/integration-tests/support/setup.js]: Doesn't the 'create'  already introduce a delay?

File: main/integration-tests/support/setup.js [hunk 2]
```diff
@@ -151,6 +160,10 @@ class Setup {
       userRole,
       projectId,
     });
+
+    // Delay to prevent eventual consistency issues from affecting the next call.
+    await sleep(1000);
```
Comment (hatimkhan) [inline main/integration-tests/support/setup.js]: Doesn't the 'create' already introduce a delay?

File: main/integration-tests/support/setup.js (hunk 1 above)
Comment (ealemayehu) [reply main/integration-tests/support/setup.js]: Thanks for catching this! I have updated the default deflake delay from 1 to 2 seconds. The 1 second delay is what the normal maximum time dynamodb needs to persist eventually consistent changes.

File: main/integration-tests/support/setup.js (hunk 2 above)
Comment (ealemayehu) [reply main/integration-tests/support/setup.js]: Fixed as explained above

**Review by hatimkhan — APPROVED** [review]
LGTM

**Pull request merged**